### DISCUSSION TITLE
Extracting tokens from a Figma file

### DISCUSSION THREAD

**Discussion by senthil-kk20** [six7/figma-tokens#131] [Q&A]
Hi,
Thank you so much for your plugin. It does wonders with our workflow. I wanted to keep a Figma design source as a source of truth and extract json from the file. Right now, I don't have clear idea on how to extract Shadows, Sizing, Spacing, Border radius, Border width and Opacity. Also, do yo have any plans to integrate Git along with JsonBin?

Thanks,
Senthil

**Comment by six7** [comment]
Hey @senthil87 - right now there is no automated way to extract design decisions for shadows, sizing, spacing, border radius, border width and opacity from a Figma file, you'd have to create these manually for now.

For the future I'd like a flow that is similar to the "Import Styles" flow, the plugin could iterate over all layers, extract tokens used and give you the possibility to name these tokens. In a way, this becomes somewhat of a design linter as well, with the added benefit that tokens might already get applied. However, this functionality is not in yet.

Regarding Git: I opened an issue so you can keep track of it, I do have plans for that, but no ETA on it yet. #134
